packages/components/Button/src/Button.types.ts

```diff
@@ -93,7 +94,7 @@ export interface IButtonProps extends Omit<IPressableProps, 'onPress'> {
   /*
    * Source URL or name of the icon to show on the Button.
    */
-  icon?: string;
+  icon?: number | string |IconProps;
```

Comment:
Given this appears twice it might be worth creating an IconPropType and using that in each location.

Reply:
Added now